llarp/handlers/tun.cpp

```diff
@@ -118,6 +118,17 @@ namespace llarp
           m_PublishIntroSet = false;
           LogInfo(Name(), " setting to be not reachable by default");
         }
+        else if(IsTrueValue(v))
+        {
+          m_PublishIntroSet = true;
+          LogInfo(Name(), " setting to be reachable by default");
+        }
+        else
+        {
+          LogError(Name(), " config option reachable = '", v,
+                   "' does not make sense");
+          return false;
+        }
       }
```

Comment:
This is more of an observation, but we now have >= 15 lines of code to handle the 'reachable' config option... Seems like we may be missing some DRY (etc.) somewhere